src/components/Schedule/AppointmentModal.tsx

```diff
@@ -85,14 +122,37 @@ export const AppointmentModal = (props: AppointmentModalProps) => {
             value: prof.displayName,
           };
         });
-        console.log(profNames);
         setProfessorOptions(profNames);
       } else {
         console.log(error);
       }
     }
   }, [data, error, loading]);
 
+  useEffect(() => {
+    if (!coursesLoading) {
+      if (!coursesError && coursesData) {
+        const queryResult = coursesData.survey.courses;
+        const termData = queryResult.filter(
+          (course: CourseID) => course.term === courseData.term
+        );
+        const courses = termData.map((course: CourseID) => {
+          return {
+            label: course.subject + " " + course.code,
+            value: {
+              code: course.code,
+              subject: course.subject,
+              title: course.title,
+            },
+          };
+        });
+        setCoursesOptions(courses);
+      } else {
+        console.log(coursesError);
+      }
```

Comment:
Would this be an error that we want to surface to the user or just ignore? Either way we should probably clean up the console log before merging